controllers/testdata/mcad_test_cases/case_1.yaml

```diff
@@ -2,4 +2,11 @@ apiVersion: codeflare.codeflare.dev/v1alpha1
 kind: MCAD
 metadata:
   name: blank-custom-resource
-spec: {}
+spec:
```

Comment:
We should create a new test case that has the controller resources defined.

Reply:
@anishasthana we should keep this test case as is, and add a second that includes the resource definitions? Right?